test/Conversion/StandardToHandshake/test_remove_block.mlir

```diff
@@ -0,0 +1,31 @@
+// NOTE: Assertions have been autogenerated by utils/update_mlir_test_checks.py
+// RUN: circt-opt -create-dataflow -remove-block-structure %s | FileCheck %s
```

Comment:
I think you should start with the IR after running -create-dataflow, rather than running it as part of the test.

Reply:
Make sense.